src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -21,6 +21,31 @@ define(function (require, exports, module) {
         linterManager      = require("linterManager"),
         pluginManager      = require("pluginManager");
 
+        var preferencesID = "interactive-linter";
+        
+        var defaultPreferences = { 
+        delay: 500,
+        enabled: false,
+        webworker: true,
+        javascript: "eslint" 
```

Comment:
This is wrong.  If you look above at `.brackets.json` you'll see that it's an array of strings.